src/dev/flang/ast/Call.java

```diff
@@ -1431,7 +1431,11 @@ private AbstractType adjustThisTypeForTarget(AbstractType t)
       {
         var inner = new Type(calledFeature().selfType(),
                              _target.typeForCallTarget());
-        t = t.replace_this_type_by_actual_outer(inner);
+        var t0 = t;
+        var c = Errors.count();
```

Comment:
What is the purpose of this `Errors.count()`?

Reply:
forgotten debug code, removed in my latest update.